tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -2123,26 +2123,47 @@ public function readFileInServerRoot($path) {
 		$this->setResponse($response);
 	}
 
+	/**
+	 * send request to list a server file
+	 *
+	 * @param string $path
+	 *
+	 * @return void
+	 */
+	public function listTrashbinFileInServerRoot($path) {
+		$response = OcsApiHelper::sendRequest(
+			$this->getBaseUrl(),
+			$this->getAdminUsername(),
+			$this->getAdminPassword(),
+			'GET',
+			"/apps/testing/api/v1/dir?dir={$path}"
+		);
+		$this->setResponse($response);
+	}
+
 	/**
 	 * @Then the file :path with content :content should exist in the server root
 	 *
 	 * @param string $path
 	 * @param string $content
 	 *
-	 * @return void
+	 * @return string
 	 */
-	public function theFileWithContentShouldExistInTheServerRoot($path, $content) {
+	public function theFileWithContentShouldExistInTheServerRoot($path, $content, $trash = false) {
 		$this->readFileInServerRoot($path);
 		Assert::assertSame(
 			200,
 			$this->getResponse()->getStatusCode(),
 			"Failed to read the file {$path}"
 		);
-
 		$fileContent = HttpRequestHelper::getResponseXml($this->getResponse());
 		$fileContent = (string) $fileContent->data->element->contentUrlEncoded;
 		$fileContent = \urldecode($fileContent);
 
+		if($trash){
```

Comment:
```suggestion
		if ($trash){
```
https://drone.owncloud.com/owncloud/core/24035/2/6
code style is failing

Reply:
It should be fixed by now